types/jpush-react-native/index.d.ts

```diff
@@ -0,0 +1,120 @@
+// Type definitions for jpush-react-native 1.6
+// Project: https://github.com/jpush/jpush-react-native
+// Definitions by: HuHuanming <https://github.com/huhuanming>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default class JPush {
+    /**
+     * Android only
+     * 初始化JPush 必须先初始化才能执行其他操作
+     */
+    static initPush(): void;
+
+    /**
+     * Android
+     */
+    static stopPush(): void;
+
+    /**
+     * Android
+     */
+    static resumePush(): void;
+
+    /**
+     * Android
+     */
+    static clearAllNotifications(): void;
+
+    /**
+     * Android
+     */
+    static clearNotificationById(id: string): void;
+
+    /**
+     * Android
+     */
+    static getInfo(cb: (map: any) => void): void;
+
+    static setTags(tags: string[], successCallback: (resultCode: number[]) => void, failedCallback: () => void): void;
```

Comment:
From the source, it looks like `successCallback` never gets called with parameters.